developer_setup.md

```diff
@@ -76,8 +76,8 @@ ManageIQ requires a memcached instance for session caching and a PostgreSQL data
    | ---------- | --- |
    | dnf        | `sudo dnf -y install postgresql-server` |
    | yum        | `sudo yum -y install postgresql-server` |
-   | apt        | `sudo apt -y install postgresql-11` |
-   | brew       | `brew install postgresql@11` |
+   | apt        | `sudo apt -y install postgresql-13` |
```

Comment:
Do we want to leave this as `sudo apt -y install postgresql` ?

Reply:
yeah, I'd rather we specify the versions once at top and give the general instructions down below.  These specific things always get stale so quickly.

Reply:
Yeah I thought we wanted to use the virtual package `postgresql` rather than specific versions